src/work/work_trsmA.cc

```diff
@@ -252,11 +245,7 @@ void trsmA(Side side, scalar_t alpha, TriangularMatrix<scalar_t> A,
                     bcast_list_upd_B.push_back(
                         {k, j, { A.sub(k + 1, mt - 1, k, k), }});
                 }
-
-                auto B_row_k = B.sub( k, k, 0, nt-1 );
-                // XXX Should it be just 1 at the last iteration?
-                // XXX Should we ignore this since the life will be removed
-                B.template listBcast<target>( bcast_list_upd_B, layout, k, lookahead + 1 );
```

Comment:
Note: tag=k, life_factor=lookahead + 1
That life_factor doesn't make sense, but it's being removed now.